othello/templates/emails/winner.html

```diff
@@ -1,5 +1,6 @@
+{% load humanize %}
 <h4 style="color: #33a122">
-    Congratulations {{ name }}! You have placed {{ rank }} in the past Othello AI Tournament!
+    Congratulations {{ name }}! You have placed {{ rank|ordinal }} in the past Othello AI Tournament!
```

Comment:
```suggestion
    Congratulations {{ name }}! You have placed {{ rank|ordinal }} in the previous Othello AI tournament!
```

I think previous or last would work; I don't think "past" is really appropriate